cannot find type `Vec` in this scope

Using version 0.2 of pwasm-abi-derive like this:

```
#[eth_abi(TokenEndpoint)]
pub trait TokenInterface {
    /// The constructor
    fn constructor(&mut self, _total_supply: U256);
    /// Total amount of tokens
    fn totalSupply(&mut self) -> U256;
}
```

Will result in the following error on build (using nightly-2018-11-12):

```
error[E0412]: cannot find type `Vec` in this scope
```

The following line in lib.rs of pwasm-abi-derive seems to be the culprit:
```
use pwasm_abi::types::{H160, H256, U256, Address};
```

Changing it to include "Vec":
```
use pwasm_abi::types::{H160, H256, U256, Address, Vec};
```
fixes the build.

--- a/derive/src/lib.rs
+++ b/derive/src/lib.rs
@@ -171,7 +171,7 @@ fn generate_eth_endpoint_wrapper(
 		mod #mod_name_ident {
 			extern crate pwasm_ethereum;
 			extern crate pwasm_abi;
-			use pwasm_abi::types::{H160, H256, U256, Address};
+			use pwasm_abi::types::{H160, H256, U256, Address, Vec};
 			use super::#name_ident_use;
 			#endpoint_toks
 		}
@@ -205,7 +205,7 @@ fn generate_eth_endpoint_and_client_wrapper(
 		mod #mod_name_ident {
 			extern crate pwasm_ethereum;
 			extern crate pwasm_abi;
-			use pwasm_abi::types::{H160, H256, U256, Address};
+			use pwasm_abi::types::{H160, H256, U256, Address, Vec};
 			use super::#name_ident_use;
 			#endpoint_toks
 			#client_toks

